coalib/bearlib/languages/definitions/SASS.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class SASS:
+    extensions = '.sass',
+    comment_delimiters = '//',
+    multiline_comment_delimiters = {'/*': '*/'}
+    encapsulators = {'(': ')', '[': ']'}
+    versions = {3.1, 3.2, 3.3, 3.4, 3.5, 3.6, 4.0}
```

Comment:
Where did you get this list from?

SASS didnt start at version 3.1 did it?

This should be versions of the languages, not versions of an implementation in one language.

Reply:
Oh sorry I was following the code like in SCSS.py. Can you give me a link where the versions are listed. 

I got the list from here - https://rubygems.org/gems/sass/versions